app/models/account.rb

```diff
@@ -2,6 +2,12 @@
 
 # Customer organization account
 class Account < ApplicationRecord
+  scope :not_cross_search_tenants, -> {where('settings @> ?', {shared_search: 'false'}.to_json)}
+  has_many :children, class_name: "Account", foreign_key: "parent_id", dependent: :nullify, inverse_of: :parent
+  belongs_to :parent, class_name: "Account", inverse_of: :parent, foreign_key: "parent_id", optional: true
```

Comment:
what does this parent / child thing do in your repo? I see you adding the infra here but it doesn't seem like the whole feature is happening? I'm maybe ok with that, but I would like to understand what the end goal is for it if possible.

Reply:
A search tenant is  a parent of all the tenants in its solr collection alias. So it allows us to do somethings:
1. Query for  the tenants in a particular shared search and check those in the edit page
2.  For Ubiquity we  have  a tab next to contacts which is called 'shared search; which points to the parent or the shared search a particular tenant is in.  when you click it, it redirect to the shared search page..